packages/appstore/src/components/modals/open-positions-svg-modal/open-positions-svg-modal.scss

```diff
@@ -0,0 +1,8 @@
+.open-positions-svg {
+    &__modal-title {
+        margin-bottom: 2.4rem;
+    }
+    &__modal-footer {
+        padding: 0 2.4rem 2.4rem !important;
```

Comment:
can we accomplish this in any other way, avoiding `!important` usage? 🗝️

Reply:
I'm afraid not. Modal footer is padding according to figma is `24px` and default settings are `16px`. Thus, the need to use `!imp` to override.